lib/types/mutable-long.js

```diff
@@ -79,6 +79,10 @@ MutableLong.fromString = function fromString(str, radix) {
   return result;
 };
 
+MutableLong.toString = function toString() {
```

Comment:
I think this is doable, but it will be challenging given that i'm not very familiar with the `MutableLong` code (and I'm not very good at bit-twiddling).  Will likely save this towards the end.

Reply:
Probably you meant `MutableLong.prototype.toString` :)
I think we could just call `this.toImmutable().toString()`. Otherwise we will have to implement division operation which is somehow complex.

Reply:
Hah!  Yeah, that would be....much easier 😄 .   Definitely will go that route.